.eslines.json

```diff
@@ -0,0 +1,11 @@
+{
+    "branches": {
+        "default": ["downgrade-unmodified-lines"]
+    },
+    "processors": {
+        "downgrade-unmodified-lines": {
+            "remote": "origin/master",
+            "rulesNotToDowngrade": ["no-unused-vars"]
```

Comment:
Why this rule is different?

Reply:
Via https://www.npmjs.com/package/eslines#how-to-use-it:
> _"The `no-unused-vars` rule won't be downgraded, though - this is one case where changing one line can cause a linting error in a different one so we recommend preventing it from downgrading."_